packages/core/core/src/serializer.js

```diff
@@ -5,7 +5,7 @@ import {serializeRaw, deserializeRaw} from './serializerCore';
 export {serializeRaw, deserializeRaw} from './serializerCore';
 
 const nameToCtor: Map<string, Class<any>> = new Map();
-const ctorToName: Map<Class<any>, string> = new Map();
+export const ctorToName: Map<Class<any>, string> = new Map();
```

Comment:
```suggestion
const ctorToName: Map<Class<any>, string> = new Map();
```

don't need to export that anymore